.travis.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash -eu
 
 bundle exec jekyll build --strict_front_matter
+bundle exec htmlproofer _site --url-swap "http?s?\:\/\/moveit\.ros\.org\/:"
```

Comment:
lets move this change into a separate PR and deal with the Travis errors there, so we can get this merged in now

Reply:
How do I get past the travis errors here... it won't let me merge it when they are failing.